pkg/cache/cache_test.go

```diff
@@ -1449,15 +1481,18 @@ func TestClusterQueueUsage(t *testing.T) {
 					t.Fatalf("Workload %s was not added", workload.Key(&w))
 				}
 			}
-			resources, workloads, err := cache.Usage(tc.clusterQueue)
-			if err != nil {
-				t.Fatalf("Couldn't get usage: %v", err)
-			}
-			if diff := cmp.Diff(tc.wantUsedResources, resources); diff != "" {
-				t.Errorf("Unexpected used resources (-want,+got):\n%s", diff)
-			}
-			if workloads != tc.wantWorkloads {
-				t.Errorf("Got %d workloads, want %d", workloads, tc.wantWorkloads)
+			// We must check whether the same order of 'flavorUsage' is returned.
+			for i := 0; i < 10; i++ {
```

Comment:
Do we need to test this many times? We already run the entire suite of unit tests 3 times. I suspect that flakiness would be enough.

Reply:
It makes sense.
Done.